libbpfgo.go

```diff
@@ -1,7 +1,7 @@
 package libbpfgo
 
 /*
-#cgo LDFLAGS: -lelf -lz
+#cgo LDFLAGS: -lelf -lz -lbpf
```

Comment:
For tracee, we use our own libbpf submodule (a statically compiled .a), so adding -lbpf will not work with it

Reply:
this was an error for my last push and local checks, wasn't meant to be there. thank you